content/influxdb/cloud-iox/query-data/tools/pandas.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Use pandas to analyze and visualize data
+seotitle: Use Python and pandas to analyze and visualize data
+description: >
+  Install and run [pandas](https://pandas.pydata.org/), the Python Data Analysis Library,
+  to analyze and visualize data stored in a bucket powered by InfluxDB IOx.
+weight: 101
+menu:
+  influxdb_cloud_iox:
+    parent: Analyze and visualize data
+    name: Use pandas
+influxdb/cloud-iox/tags: [analysis, pandas, pyarrow, python, visualization]
+---
+
+Use [pandas](https://pandas.pydata.org/), the Python Data Analysis Library, to process, analyze, and visualize data
+stored in an InfluxDB bucket powered by InfluxDB IOx.
+
+> **pandas** is an open source, BSD-licensed library providing high-performance,
+> easy-to-use data structures and data analysis tools for the Python programming language.
+>
+> {{% caption %}}[pandas documentation](https://pandas.pydata.org/docs/){{% /caption %}}
+
+<!-- TOC -->
+
+- [Install prerequisites](#install-prerequisites)
+- [Install pyarrow](#install-pyarrow)
+- [Install pandas](#install-pandas)
+- [Use PyArrow to convert query result data to pandas](#use-pyarrow-to-convert-query-result-data-to-pandas)
+- [Use pandas to analyze data](#use-pandas-to-analyze-data)
+    - [View information and statistics for data](#view-information-and-statistics-for-data)
+    - [Downsample time series](#downsample-time-series)
+
+<!-- /TOC -->
+
+## Install prerequisites
+
+The examples in this guide assume using a Python virtual environment and the Flight SQL library for Python.
+For more information, see how to [get started querying InfluxDB with Python and flightsql-dbapi](/influxdb/cloud-iox/query-data/execute-queries/flight-sql/python/)
+
+## Install pyarrow
+
+In order to read query results in Apache Arrow format and then convert them to pandas, you need to
+install and import the `pyarrow` bindings:
+
+- In your terminal, use `pip` to install `pyarrow` in your [Python virtual environment](/influxdb/cloud-iox/query-data/execute-queries/flight-sql/python/#create-a-project-virtual-environment):
+
+    ```sh
+    pip install pyarrow
```

Comment:
I might consider changing everything to pip3 especially since venv creates python 3 env by default.

Reply:
> I might consider changing everything to pip3 especially since venv creates python 3 env by default.

@Anaisdg When I'm using venv or conda on Mac, then I can use either:

```sh
(pydata) ja@MacBook-Pro influxdb-py % which python
/Users/ja/Documents/GitHub/influxdb-py/envs/pydata/bin/python
(pydata) ja@MacBook-Pro influxdb-py % which python3
/Users/ja/Documents/GitHub/influxdb-py/envs/pydata/bin/python3
(pydata) ja@MacBook-Pro influxdb-py % which pip
/Users/ja/Documents/GitHub/influxdb-py/envs/pydata/bin/pip
(pydata) ja@MacBook-Pro influxdb-py % which pip3
```

That was actually one of my reasons for recommending virtual environments. Do you see `python3` more often than `python` in sample code in the wild?